src/ngx_http_echo_module.h

```diff
@@ -59,7 +59,7 @@ typedef struct {
 
     /* fields "lengths" and "values" are set by
      * the function ngx_http_script_compile,
-     * iff the argument value indeed contains
+     * if the argument value indeed contains
```

Comment:
`iff` is actually a logical or mathematical slang meaning "if and only if".

Reply:
o, i learned, thanks.